Woof/Woof/Model/Preferences/PreferencesHandler.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+/**
+ A type used for managing user preferences in the application.
+ */
+enum PreferencesHandler {
+    // MARK: - Internal interface
+
+    /**
+     Saves the selected role as a user preference.
+
+     - Parameters:
+        - userRole: The selected role to be saved.
+
+     - Returns: A boolean value indicating whether the saving operation was successful.
+     */
+    @discardableResult static func set(userRole: Role) -> Bool {
+        var preferences: Preferences = loadPreferences() ?? Preferences()
```

Comment:
nit:

Please use type inferring from the compiler:

```suggestion
        var preferences = loadPreferences() ?? Preferences()
```

Reply:
fixed